File: checkPalindrome.java
import java.util.*;
class CheckPalindrome{
public static void main(String[] args){
        int a=121;
        boolean ans=countDigit(a);
        System.out.println(ans);
}
public static boolean countDigit(int a){
    int temp=a;
    int rv=0;
    while(temp!=0){
        int dig=temp%10;
        rv=rv*10+dig;
        temp=temp/10;
    }
    if(a==rv){
        return true;
    }
    return false;
}
}

File: checkPalindrome.js
let n=2552;
const checkPalindrome =(n) =>{
    let temp=n;
    let rv=0;
    while(temp!==0){
        let dig=temp%10;
        rv=rv*10+dig;
        temp=Math.floor(temp/10);
    }
    if(n===rv){
        return true;
    }
    return false;

}
console.log(checkPalindrome(n));